### PR TITLE
pass script file into filters,use in collections to set worker

### PR DESCRIPTION
**Pull request by msuhovec**

### CODE REVIEW

File: lib/rof/utility.rb [hunk 1]
```diff
@@ -23,6 +24,16 @@ def initialize
...
+    #use base directory of given file for workdir
+    def set_workdir(filename)
+      @workdir = File.dirname(filename)
+    end
+
+    #give base directory of given file for workdir
+    def get_workdir
+      @workdir
+    end
+
```
Comment (dbrower) [inline lib/rof/utility.rb]: Where is @workdir used? I don't see anyone using this directory nor do I see a call to cwd()

File: lib/rof/filters/work.rb [hunk 2]
```diff
@@ -17,7 +17,8 @@ def initialize
       end
 
       # wade through object list
-      def process(obj_list)
+      def process(obj_list, filename)
+	@utility.set_workdir(filename)
```
Comment (dbrower) [inline lib/rof/filters/work.rb]: fix indentation here.

File: lib/rof/utility.rb (hunk 1 above)
Comment (dbrower) [inline lib/rof/utility.rb]: ok. I see it used above. rename "get_workdir" to "workdir".

Would it be simpler to always change the current working directory to be where the rof file is? Then work dir would not need to be passed around?

File: lib/rof/filters/work.rb (hunk 2 above)
Comment (msuhovec) [inline lib/rof/filters/work.rb]: OK

**Pull request merged**